components/hab/src/cli.rs

```diff
@@ -566,8 +566,7 @@ pub fn get(feature_flags: FeatureFlag) -> App<'static, 'static> {
                 (@arg FORCE: --force "Skip checking availability of package and \
                     force uploads, potentially overwriting a stored copy of a package.")
                 (@arg AUTO_BUILD: --("auto-build") "Enable auto-build for all packages in this upload. Only applicable to SaaS Builder.")
-                (@arg CREATE_ORIGINS: --("create-origins") "Create any origins that do not exist in targeted Builder.")
-                (@arg ACCEPT: --("accept") "Do not prompt for confirmation of auto creation and ownership of any origins that do not exist.")
+                (@arg AUTO_CREATE_ORIGINS: --("auto-create-origins") "Do not prompt for confirmation before auto creating any origins that do not exist.")
```

Comment:
I'd suggest using a description like this: "Skip the confirmation and automatically create origins that do not exist in the target Builder."
The two "do not"  in a sentence may make it a little bit harder to understand.

Reply:
Thanks, adjusted  in https://github.com/habitat-sh/habitat/pull/7133/commits/5f5cafefb4ab581f26d69e4f59770c099d5ae378

Reply:
Awesome! Thanks so much @jeremymv2 !